src/test/java/api/requests/RequestsAPICreationTests.java

```diff
@@ -1654,8 +1654,37 @@ public void cannotCreateRequestWhenRequesterHasActiveRequestManualBlocks() {
     assertThat(postResponse, hasStatus(HTTP_UNPROCESSABLE_ENTITY));
     assertThat(postResponse.getJson(), hasErrors(1));
     assertThat(postResponse.getJson(), hasErrorWith(allOf(
-      hasMessage("Patron blocked from requesting"))));
-      hasParameter("reason", "Display description");
+      hasMessage("Patron blocked from requesting"),
+      hasParameter("reason", "Display description")))
+    );
+  }
+
+  @Test
+  public void cannotCreateRequestWhenRequesterHasActiveRequestManualBlocksAndExpirationDateIsNotSet() {
+    final IndividualResource item = itemsFixture.basedUponSmallAngryPlanet();
+    final IndividualResource requester = usersFixture.rebecca();
+    final UUID pickupServicePointId = servicePointsFixture.cd1().getId();
+
+    final UserManualBlockBuilder userManualBlockBuilder = getManualBlockBuilder()
+      .withRequests(true)
+      .withUserId(String.valueOf(requester.getId()));
+
+    final RequestBuilder requestBuilder = createRequestBuilder(item,
+      requester,
+      pickupServicePointId,
+      new DateTime(2017, 7, 22, 10, 22, 54, DateTimeZone.UTC));
```

Comment:
We can create a PAGE request and avoid check-out action. Also, we can move this part directly before `requestsClient.attemptCreate`. Example:

``` 
new RequestBuilder()
  .withId(UUID.randomUUID())
  .open()
  .page()
  .forItem(item)
  .by(requester)
  .fulfilToHoldShelf()
  .withPickupServicePointId(pickupServicePointId);
```